### PR TITLE
Feature/map 154/edit personal location

### PR DESCRIPTION
**Pull request by Yojuks**
you can change data by form-data - locationName, description, arrayPhotos

### CODE REVIEW

File: backend/controllers/LocationsController.ts [hunk 1]
```diff
@@ -94,34 +94,42 @@ const LocationsController = {
...
-          {
-            $set: fields.reduce(
-              (prev: any, curr: any) => ({
-                ...prev,
-                [curr.name]: curr.value
-              }),
-              {}
-            )
+      console.log(id, 'id');
+
```
Comment (6mahdapihka9) [inline backend/controllers/LocationsController.ts]: you should delete logs

File: backend/controllers/LocationsController.ts [hunk 2]
```diff
@@ -94,34 +94,42 @@ const LocationsController = {
...
   },
-  async changeLocationInfo(req: Request, res: Response) {
+
+  async changeLocationData(req: Request, res: Response) {
     try {
-      const { _id, fields } = req.body;
+      let { id } = req.params;
+      let { locationName, description } = req.body;
 
-      const location = await Location.findById(_id);
```
Comment (6mahdapihka9) [inline backend/controllers/LocationsController.ts]: will there be an opportunity to change location filters??
if yes - they should be passed in the body as well

**Review by 6mahdapihka9 — CHANGES_REQUESTED** [review]

File: backend/controllers/LocationsController.ts (hunk 1 above)
Comment (Yojuks) [reply backend/controllers/LocationsController.ts]: done

File: backend/controllers/LocationsController.ts (hunk 2 above)
Comment (Yojuks) [reply backend/controllers/LocationsController.ts]: done

**Review by obozhko-dev — APPROVED** [review]

**Review by dobbi-free — APPROVED** [review]

**Review by vasylneshchadimov — APPROVED** [review]

**Review by dobbi-free — APPROVED** [review]

**Review by 6mahdapihka9 — APPROVED** [review]

**Pull request merged**